packages/lesswrong/components/recommendations/LWRecommendations.tsx

```diff
@@ -182,7 +182,7 @@ const LWRecommendations = ({
             settings={frontpageRecommendationSettings}
             onChange={(newSettings) => setSettings(newSettings)}
           /> }
-        {!bookDisplaySetting.get() && <AnalyticsContext pageSubSectionContext="frontpageCuratedCollections">
+        {!bookDisplaySetting.get() && <AnalyticsContext pageSubSectionContext="spotlightItem">
```

Comment:
I think you only want either this or https://github.com/ForumMagnum/ForumMagnum/pull/9030/files#diff-6d2001ac6751cdb5af41ef589c230e3ce128790497e443c981c43fdee2f22c45R43 ?